frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js

```diff
@@ -0,0 +1,82 @@
+import React, { useContext } from "react";
+import { TbRefresh, TbCloudDownload } from "react-icons/tb";
+import { AppContext } from "../../../../../contexts/context";
+import zipModel from "../../../utils/zipModel";
+import Swal from "sweetalert2";
+
+import DropBox from "../../../../../containers/common/DropBox";
+import Button from "../../../../../containers/common/Button";
+
+const ModelSave = ({ refresh }) => {
+  const { drop, setDrop, parameters, setParameters } = useContext(AppContext);
+
+  const set_parameters = (e) => {
```

Comment:
setDrop should use another naming. If not the dropbox in hyperparameter and model save are the same.